### PR TITLE
adds missing property webhookproxy events and change namespace in Tailor file

### PR DESCRIPTION
**Pull request by stitakis**
follow up of https://github.com/opendevstack/ods-provisioning-app/issues/429
additionally changes the namespace in prov app tailor file to namespace `ods`

### CODE REVIEW

File: ods-provisioning-app/ocp-config/cm.yml [hunk 1]
```diff
@@ -237,6 +237,9 @@ objects:
       openshift.jenkins.project.name.pattern=jenkins-%s-cd%s
       openshift.jenkins.project.webhookproxy.host.pattern=webhook-proxy-%s-cd%s
 
+      # list of supported webhook events
+      openshift.jenkins.project.webhookproxy.events=repo:refs_changed,pr:merged,pr:declined,pr:deleted
```
Comment (michaelsauter) [inline ods-provisioning-app/ocp-config/cm.yml:241]: Can we extract the value to a parameter? That way one can override it - `pr:opened` is also supported, but not added by default.

The parameter should then end up in the `configuration-sample/ods-core.env.sample` file.

Comment (stitakis) [reply ods-provisioning-app/ocp-config/cm.yml]: yeah... it is added now in the 2nd commit

File: configuration-sample/ods-core.env.sample [hunk 2]
```diff
@@ -234,3 +234,5 @@ LOGGING_FILE_PATH=/opt/provision/history/logs
 # If you use or might use documentation templates in the future,
 # it is highly recommended to add "ods-document-generation-templates" to this list.
 READABLE_OPENDEVSTACK_REPOS=ods-jenkins-shared-library,ods-quickstarters
+
+WEBHOOK_PROXY_EVENTS=repo:refs_changed,pr:merged,pr:declined,pr:deleted
```
Comment (michaelsauter) [inline configuration-sample/ods-core.env.sample:238]: Cool. Could you add some description? E.g.:
```
# Comma-separated list of events for which Bitbucket sends a request to the webhook proxy service
# to create/start a Jenkins pipeline or delete a Jenkins pipeline.
# Pipelines can be created/started for: repo:refs_changed, pr:opened
# Pipelines will be deleted for: pr:merged, pr:declined, pr:deleted
```

Comment (stitakis) [reply configuration-sample/ods-core.env.sample:242]: description makes sense to me. I have included now in the 3rd commit

**Pull request merged**